src/components/NewQuestionnaire/Questions.vue

```diff
@@ -30,31 +45,37 @@
 <script lang="ts">
 import { defineComponent, ref } from 'vue'
 import { QuestionType } from '/@/lib/apis'
-import QuestionTitle from './QuestionTitle.vue'
+import QuestionnaireTitle from './QuestionnaireTitle.vue'
 import AddQuestionButtons from './AddQuestionButtons.vue'
 import { createNewQuestion, QuestionData } from './use/utils'
+import Card from '/@/components/UI/Card.vue'
 import TextForm from './Forms/TextForm.vue'
 import ChoiceForm from './Forms/ChoiceForm.vue'
 import LinearScaleForm from './Forms/LinearScaleForm.vue'
+import QuestionDispose from './Forms/QuestionDispose.vue'
+import QuestionUpdown from './Forms/QuestionUpdown.vue'
 
 export default defineComponent({
   name: 'Questions',
   components: {
-    QuestionTitle,
+    QuestionnaireTitle,
+    Card,
     TextForm,
     ChoiceForm,
     LinearScaleForm,
-    AddQuestionButtons
+    AddQuestionButtons,
+    QuestionDispose,
+    QuestionUpdown
   },
   setup() {
     const questions = ref<QuestionData[]>([])
 
-    const addQuestion = (type: string) => {
+    const addQuestion = (type: QuestionType) => {
       const question = createNewQuestion(type)
-      if (question) questions.value.push(question)
+      questions.value.push(question)
     }
 
-    const updateQuestion = (newData: QuestionData, index: number) => {
+    const updateQuestions = (newData: QuestionData, index: number) => {
```

Comment:
```suggestion
    const updateQuestions = (index: number) => (newData: QuestionData) => {
```
こうして`**Form`に`index`を渡さずに`@update="updateQuestions(i)"`にすると責任の範囲が分かりやすくなる気がします